dev-tools/package.json

```diff
@@ -11,6 +11,7 @@
     "stylelint-config-recommended": "^7.0.0",
     "stylelint-config-standard": "^25.0.0",
```

Comment:
These two other `stylelint-config` packages should be able to be removed now, `stylelint-config-standard-scss` includes the ones that it needs.